src/hr_alg3.rs

```diff
@@ -23,7 +22,6 @@ pub struct Hr {
     peak_flag : u8,
     wild_flag : u8,
     above_pts : ConstGenericRingBuffer<i32, ABOVE_SIZE>,
```

Comment:
Why are these i32 and not u32 or u16?

Reply:
Yeah, I need to do something about that. Comment more in the struct? Thinking...